PowerEditor/src/Parameters.h

```diff
@@ -849,7 +855,9 @@ struct NppGUI final
 	std::wstring _definedSessionExt;
 	std::wstring _definedWorkspaceExt;
 
+	// items with no Notepad++ GUI to set
 	std::wstring _commandLineInterpreter = CMD_INTERPRETER;
+	int _registerForOSAppRestart = DEFAULT_OS_APP_RESTART_FLAGS; // -1 ... disable (unregister)

```

Comment:
Why do we need `_registerForOSAppRestart`? After all, it's a argument which contains flags...
We can pass this argument directly while calling `RegisterApplicationRestart`.

Reply:
Because of user can fine adjust the OS Restart Manager setting (in my current PR variant with config.xml, where another set of these restart flags can be stored...).